activerecord/lib/active_record/table_metadata.rb

```diff
@@ -19,7 +19,7 @@ def type(column_name)
     end
 
     def has_column?(column_name)
-      klass&.columns_hash.key?(column_name)
+      klass&.columns_hash&.key?(column_name)
```

Comment:
:+1:

Out of curiosity, is this exercised by this change, or just noticed in passing?

Reply:
Ah, this used to be exercised in the previous version of this PR, but isn't anymore. I'll remove it to make the PR more focused.